Kitodo/src/main/java/org/kitodo/workflow/Solution.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.workflow;
+
+public class Solution extends Message{
```

Comment:
Missing space between `Message` and `{`.

Reply:
Done.